### DISCUSSION TITLE
Is there any plugin in Vscode for developing with lit ?

### DISCUSSION THREAD

**Discussion by Linya-IronMan** [lit/lit#2649] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/77786952/159111868-08678ae9-9576-4a80-a764-d9513616e4de.png)

style code has no syntax highlighting, the development experience is not good

**Comment by abdonrd** [comment, marked as answer]
https://marketplace.visualstudio.com/items?itemName=runem.lit-plugin

**Comment by christophe-g** [comment]
https://marketplace.visualstudio.com/items?itemName=runem.lit-plugin is a great plugin 

Just be aware that it currently breaks auto from suggestion for now: 
https://github.com/runem/lit-analyzer/issues/200#issuecomment-1064408628

ccing @rictic or @justinfagnani for as they are very busy people and some folks hope it will be resolved soon ; )

**Reply by kevinpschaaf** [reply]
Thank you! We're aware and actively working on getting a fix for lit-plugin out soon. Thank you for your patience.

**Reply by rictic** [reply]
Updated version is out, with faster bootup and a fix for auto import from suggestion
